_pages/views.md

```diff
@@ -172,6 +172,269 @@ vbox.
 ![Containers Example 1](../assets/images/views/ContainerExample1.PNG 
 "Example containers with 1 hbox and 2 vboxes")
 
+Borders and Border Titles on the forms can help further separate and
+distinguish each form 
+
+```xml
+
+<!-- first form (left vbox) -->
+<form border="true" borderTitle="Left vbox"> 
+
+<!-- second form (right vbox) -->
+<form border="true" borderTitle="Right vbox"> for the right vbox
+
+```	
+	
+These borders and titles produce the following view:
+	
+![Containers Example 2](../assets/images/views/ContainerExample2.PNG 
+"Example containers with 1 hbox and 2 vboxes, with added borders and titles")
+
+If ‘Text field 2’ and ‘Date field 2’ are required, we can make them required one of 
+two ways. As Text field 2 is always required, we can make it required within the 
+‘DocumentName.xml’ (need to mention this file name earlier) file like so:
+	
+```xml
+	
+<text name="text2" required="true">
+	<displayName>Text field 2</displayName>
+    	<description>This is our second text field</description>
+  	<length>20</length>
+</text>
+	
+```	
+
+And as ‘Date field 2’ is only required when being used in this view, we can make 
+it required within the edit view as shown:
+
+```xml
+	
+<item labelAlign="left" required="true">
+	<default binding="date2"/>
+</item>
+
+```
+
+If we decide that ‘Time field 2’ should be renamed, we can change the Display Name 
+inside 'DocumentName.xml', or we can use a label inside the ‘time2’ binding within the edit 
+view this like: 
+
+```xml
+
+<item labelAlign="left" label="SECOND TIME FIELD">
+	<default binding="time2"/>
+</item>
+	
+```
+
+These recent changes have altered the edit view like so:
+
+![Required/Renamed Fields](../assets/images/views/RequiredRenamedFields.PNG "Renamed and Required fields")
+
+
+#### Containers Example #2 - Two side by side vboxes and one hbox underneath
+
+![Containers Example 2 Wireframe](../assets/images/views/ViewsPic2.PNG 
+"Wireframe example of 2 vboxes with one hbox underneath")
+
+The above Container Layout is acheived through a pair of vboxes 
+inside a hbox, a second hbox is then added beneath to make room for 
+a third form:
+
+```xml
+
+<hbox>
+	Top two vboxes in here
+</hbox>
+<hbox>
+	<form border="true" borderTitle="Third form">
+		<column responsiveWidth="2"/>
+		<column responsiveWidth="4"/>
+		<column responsiveWidth="2"/>
+		<column responsiveWidth="4"/>
+		<row>
+			<item labelAlign="left">
+				<default binding="text3"/>
+			</item>
+			<item labelAlign="left">
+				<default binding="boolean2"/>
+			</item>
+		</row>
+		<row>
+			<item labelAlign="left">
+				<default binding="time3"/>
```

Comment:
In order for your condition defined below to update the visibility of `dateTime2`, this needs to define a change handler with a re-render. This one probably needs to be changed in the example to:

```xml
<textField binding="time2">
	<onChangedHandlers>
		<rerender clientValidation="false" />
	</onChangedHandlers>
</textField>
```